### ISSUE TITLE
Feature Request: Multiple docker network

### ISSUE THREAD

**Issue by kamcio96** [pterodactyl/panel#1486]
With multiple selection in panel (like ip mapping, primary and others networks)

https://docs.docker.com/engine/reference/commandline/network_connect/

**Comment by DaneEveritt** [comment]
Yeah I'd like to completely re-approach the network setup and allow all of a user's server to be connected under a single network on the machine, which would then completely remove the ability for other users on the machine to access said server's from their instances as well.

**Comment by DaneEveritt** [comment]
In the interest of maintainer sanity and cleaning up an enormous backlog of stale feature requests we have moved to requiring all feature requests be discussed within [Discussions](https://github.com/orgs/pterodactyl/discussions/categories/feature-requests). Please consider re-opening this request there if you believe it is still relevant and worth consideration.

See https://github.com/pterodactyl/panel/issues/5491 for additional context.

Thanks.

<sub>_This comment is in the interest of issue tracker maintenance: it does not imply that maintainers are endorsing or dismissing your request._</sub>
